xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
+        <value>The cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+This property gets the cipher suite that is going to be used in the communication between a client and server by the underlying SSL/TLS implementation. This property is only available after a successful call to <xref:System.Net.Security.SslStream.AuthenticateAsServer%2A> or <xref:System.Net.Security.SslStream.AuthenticateAsClient%2A>.
```

Comment:
There's a tense problem with this... data may have already been transferred, the cipher suite may already be in use.  I'd just delete the first sentence.

```suggestion
This property is only available after a successful call to <xref:System.Net.Security.SslStream.AuthenticateAsServer%2A> or <xref:System.Net.Security.SslStream.AuthenticateAsClient%2A>.
```

(I don't know if it needs to pedantically talk about the Async variants as well)

Reply:
yes, I think everything after client/server hello will be encrypted (minus some edge cases around re-negotiation and stuff)